integration_tests/sdk/setup_integration.py

```diff
@@ -163,6 +163,9 @@ def _setup_external_sqlite_db(path: str):
 def _setup_postgres_db():
     _execute_command(["aqueduct", "install", "postgres"])
 
+def _setup_maria_db():
+    _execute_command(["aqueduct", "install", "mariadb"])
```

Comment:
What does this do?

I made changes to make both the mysql and mariadb commands be the same.

Reply:
Oh right, I can combine them.